rust/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# To be run by someone with push rights to gh-pages
+# TODO: automate this...
```

Comment:
I reckon we could add a Github actions workflow this later

Reply:
yeah, I tried in the past, and struggled, ill let someone else with more experience with github actions try.